src/Containers/ProductListings/Containers/ProductListingTable.tsx

```
@@ -58,12 +57,35 @@
+		const stickyScrollRef = useRef<ScrollView>(null);
+		const mainScrollRef = useRef<ScrollView>(null);
+		const isStickyScrolling = useRef(false);
+		const isMainScrolling = useRef(false);
+
 		// Calculate column widths once based on the data and headers
 		const columnWidths = useMemo(
 			() => calculateColumnWidths({ head, data }),
 			[head, data],
 		);
 
+		const handleStickyScroll = (event: any) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)